src/services/scheduler.ts

```diff
@@ -57,23 +57,27 @@ export async function start(dbId: string, dbUser:string) {
             break;
           }
         }
-        // once the operation completes updating the `end_date`
-        // will complete the polling
         try {
-          console.log('start job')
           let output = await promise;
-          console.log('job done, start query')
+          // once the operation completes updating the `end_date`
+          // will complete the polling
+          const query = `
+            update iasql_operation
+            set end_date = now(), output = '${output}'
+            where opid = uuid('${opid}');
+          `;
+          console.log(query);
```

Comment:
Nit: Delete this?

```suggestion
```

Reply:
so this was super useful debugging when there was an error in the query itself. I would prefer to leave it, but can remove. we definitely need to add a proper logger